platform/sync-installer/sync-installer-impl/src/main/java/org/codice/ddf/sync/installer/impl/SynchronizedInstallerImpl.java

```diff
@@ -486,6 +477,10 @@ protected ServiceTracker getServiceTracker(BundleContext bundleContext, Filter f
     return new ServiceTracker<>(bundleContext, filter, null);
   }
 
+  protected SynchronizedConfigurationListener getConfigListener(String pid) {
```

Comment:
✏️ This seems visible only for testing. Make it package private and annotate with `@VisibleForTesting`

Reply:
❓ Is there a reason this is `protected` and not package-private?